src/modules/list.node.js

```diff
@@ -0,0 +1,50 @@
+class ListHTML {
+  constructor(api) {
+    this.api = api;
+    this.node = document.querySelector('#list');
+    this.loader = document.querySelector('.loader');
+    this.listItems = document.querySelectorAll('#list');
+    this.gamer = {
+      name: document.querySelector('#name').value,
+      score: document.querySelector('#score').value,
+    };
+
+    document.querySelector('#refresh').addEventListener('click', () => {
+      this.reLoad();
+    });
+  }
+
+  async reLoad() {
+    this.listItems.forEach((element) => { element.style.display = 'none'; });
+    this.loader.style.display = 'block';
+
+    const list = await this.api.getList();
+    if (!list.length) { return; }
+
+    this.#removeAllChildNodes();
+
+    list.forEach((element) => {
+      const { user, score } = element;
+      this.#addScore(user, score);
+    });
+
+    this.loader.style.display = 'none';
+    this.listItems.forEach((element) => { element.style.display = 'block'; });
+  }
+
+    #addScore(name, score) {
+    const li = document.createElement('li');
+    li.innerHTML = `${name}:  ${score}`;
+    li.setAttribute('id', 'li');
```

Comment:
- [ ] When trying to add a score, the name and score from the input fields are not being passed in the post request. Kindly fix it
-
<img width="940" alt="image" src="https://user-images.githubusercontent.com/37901469/209135721-9cb4d345-334b-4627-a12f-3e9a5ae1a68e.png">

Reply:
Thank you, I have fixed it